frontend/src/js/app.js

```diff
@@ -1,11 +1,13 @@
 import Vue from 'vue';
+import '@mdi/font/css/materialdesignicons.css'
 import vuetify from './plugins/vuetify';
-import App from './components/App';
+import router from './plugins/router';
+import App from '../js/App.vue';
+export const bus = new Vue();
```

Comment:
for what purpose is the constant variable `bus`?

Reply:
Oh I understand. Instead of using `bus`, how about using the `this` keyword?

Reply:
Are you talking about the custom event?

Reply:
Yes. I think the `this` keyword suffice for that.

Reply:
Oh okay. But are you familiar with Event Bus in Vue?

Reply:
I'm not. But I remember using similar method in my previous project. So I don't know if it's Event Bus.

Reply:
I used it like this in my previous project:

`window.Fire = new Vue();`

then I changed it by using the `this` keyword

Reply:
I think that is same with EventBus. But i cant find any reference regarding on what is window.fire about.

Reply:
I think i need to move my bus variable in App.vue

Reply:
> I think that is same with EventBus. But i cant find any reference regarding on what is window.fire about.

I still the `this` keyword is sufficient

Reply:
Okay i will try it.

Reply:
"this" keyword not working. nothing happening